packages/react-components/react-charts-preview/package.json

```diff
@@ -69,7 +69,8 @@
     "@types/react": ">=16.14.0 <19.0.0",
     "@types/react-dom": ">=16.9.0 <19.0.0",
     "react": ">=16.14.0 <19.0.0",
-    "react-dom": ">=16.14.0 <19.0.0"
+    "react-dom": ">=16.14.0 <19.0.0",
+    "@fluentui/react": "^8.118.0"
```

Comment:
remove fluentui/react dependency. Why is it needed

Reply:
removed